src/dune/args.py

```diff
@@ -125,7 +126,8 @@ def __init__(self):
                                        'needed for core contract and deploys core, token, '
                                        'and multisig contracts')
         self._parser.add_argument('--send-action', nargs=4, action=fix_action_data,
-                                  metavar=["ACCOUNT", "ACTION", "DATA", "PERMISSION"],
+                                  metavar=["ACCOUNT", "ACTION",
```

Comment:
This probably should not have been split into multiple lines. We have pylint set to linewidth of 140-ish in main.